frontend/src/utils/URL.ts

```diff
@@ -28,7 +28,11 @@ export const getFromQuery = ({
 
 export const buildURL = (baseUrl: string, relativePath: string): string => {
```

Comment:
Thank you @devamitranjan 👏  can you please copy over the same function to the api side : api/src/utils/helpers/URL.ts